browser/src/react-redux/App/Signup/Form.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Form = props => {
+  return (
+    <div className="form">
+      <label className="form__label">
+        {props.label}
+        <span className="form__error">{props.error}</span>
+      </label>
+      <input
+        type={props.type}
+        name={props.name}
+        onChange={e => props.handleChange(props.name, e.target.value)}
```

Comment:
This anonymous function will be re-created with each re-render. Let's place this in a method above.